styles/old/layout/main.css

```diff
@@ -1,19 +1,33 @@
 #main {
   .container {
-    padding: 0 20px;
+    margin: 0 auto;
+    display: block;
+  }
+
+  .has-side-nav.container {
+    display: grid;
+    grid-template-columns: fit-content(300px) 1fr;
```

Comment:
```suggestion
    grid-template-columns: 300px 1fr;
```
For desktop resolution, it may be better to keep this area static;

<img width="999" alt="image" src="https://github.com/nodejs/nodejs.org/assets/14062599/d09deb6a-926b-4466-8b75-ee4b3c189fdb">